moml/models/mgnn/evaluation/metrics.py

```
@@ -35,347 +60,625 @@
+            probas_for_auc = flat_preds
+        else:
+            pred_labels = flat_preds.astype(int)
+            
+    elif pred_values.ndim == 2 and pred_values.shape[1] == 2:
+        # Shape (N, 2) - binary class probabilities
+        pred_labels = np.argmax(pred_values, axis=1)
+        # Use probability of positive class (assume class 1 if present)
+        if 1 in unique_classes:
+            probas_for_auc = pred_values[:, 1]
+        else:
+            probas_for_auc = pred_values[:, 1]  # Use second column as default
```

Ruff (0.12.2)

Use ternary operator `probas_for_auc = pred_values[:, 1] if 1 in unique_classes else pred_values[:, 1]` instead of `if`-`else`-block

(SIM108)